### PR TITLE
Typescript for list stories

### PR DESCRIPTION
**Pull request by wenche**
Part of #735

### CODE REVIEW

File: libraries/core-react/src/List/List.tsx [hunk 1]
```diff
@@ -41,6 +41,6 @@ const List = forwardRef<HTMLUListElement | HTMLOListElement, Props>(
   },
 )
 
-// List.displayName = 'List'
+List.displayName = 'List'
```
Comment (mimarz) [inline libraries/core-react/src/List/List.tsx:44]: Is `displayName` needed again?

Comment (wenche) [reply libraries/core-react/src/List/List.tsx:44]: Hmm, no, I was just investigating another bug. 👍

**Review by mimarz — CHANGES_REQUESTED** [review]
Setting `displayName` seems unnecessary?

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**